server/tests/articleModels.test.js

```diff
@@ -1,30 +1,33 @@
-import faker from 'faker';
 import { expect } from 'chai';
 import db from '../models';
 
 
 const validArticle = {
-  title: faker.random.word(),
-  body: faker.random.words(),
-  description: faker.random.words(),
-  slug: faker.random.word()
+  title: 'When you love',
+  body: 'body exercise is good and better',
+  description: 'Physical and health',
+  slug: 'When-you-love',
+  readTime: 3
 };
 
 const noTitle = {
-  body: faker.random.words(),
-  description: faker.random.words()
+  body: 'body exercise is good and better',
+  description: 'Physical and health',
+  readTime: 3
 };
 
 const noBody = {
-  title: faker.random.words(),
-  description: faker.random.words(),
```

Comment:
Faker library is causing irregularity on the test case, so I decided to replace it with plain text.